.github/workflows/release.yml

```diff
@@ -61,9 +61,9 @@ jobs:
         make archive
         make zip
 
-     - name: Create GH Release
-       id: create_release
-       uses: release-drafter/release-drafter@v5
+    - name: Create GH Release
+      id: create_release
+      uses: release-drafter/release-drafter@v5
 
```

Comment:
Need to combine L64 to L76 as one job. We basically added name and id for the existing step.